frontend/src/api/djangoAPI.js

```diff
@@ -2,6 +2,6 @@
 import axios from 'axios';
 
 export default axios.create({
-    //baseURL: 'http://localhost:1337/streaming/',
-    baseURL: '/streaming/',
+    baseURL: 'http://localhost:1337/streaming/',
+    //baseURL: '/streaming/',
```

Comment:
We should really find a convenient solution for automatically changing the API end point (e.g. [here](https://medium.com/@trekinbami/using-environment-variables-in-react-6b0a99d83cf5)?).
Using and environment variable for production and having a default one that is used in dev would be nice.